graphql.go

```diff
@@ -167,6 +177,10 @@ func (s *Schema) Exec(ctx context.Context, queryString string, operationName str
 func (s *Schema) exec(ctx context.Context, queryString string, operationName string, variables map[string]interface{}, res *resolvable.Schema) *Response {
 	doc, qErr := query.Parse(queryString)
 	if qErr != nil {
+		if s.collector != nil {
+			s.collector.Count("graphql.parse_error", float64(1))
+		}
+
```

Comment:
This repo is the fork and I'm pretty they are not happy to add our `stats.Collector` as a dependency :).

So this should be done on caller side by checking response for `QueryError`-s

Reply:
reverted